packages/manager-components/src/components/content/price/price.component.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TEXT_COLOR_HUE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import {
+  OvhSubsidiary,
+  ovhSubsidiaryToCurrency,
+  CentsUnit,
+  Frequencytype,
+} from '../../../enumTypes';
+
+export interface PriceProps {
+  price: number;
+  frequency?: Frequencytype;
+  unit?: CentsUnit;
+  htOnly?: boolean;
+  withTTC?: boolean;
+  includeGST?: boolean;
+  excludeGST?: boolean;
+  htWithTTC?: boolean;
+  ovhSubsidiary?: OvhSubsidiary;
+}
+
+export interface PriceOptions {
+  htOnly?: boolean;
+  withTTC?: boolean;
+  includeGST?: boolean;
+  excludeGST?: boolean;
+  htWithTTC?: boolean;
+  ovhSubsidiary?: OvhSubsidiary;
+}
+
+function getConvertedPrice(price: number, unit: CentsUnit): number {
+  switch (unit) {
+    case CentsUnit.microcents:
+      return price / 100000000;
+    case CentsUnit.millicents:
+      return price / 1000;
+    case CentsUnit.cents:
+      return price / 100;
+    default:
+      return price;
+  }
+}
+
+function formatCurrency(price: string, ovhSubsidiary: OvhSubsidiary): string {
+  const currency = ovhSubsidiaryToCurrency[ovhSubsidiary] || 'EUR';
+  return new Intl.NumberFormat(ovhSubsidiary, {
+    style: 'currency',
+    currency,
+  }).format(parseFloat(price));
+}
+
+function addFrequencyLabel(
+  formattedPrice: string,
+  frequency: Frequencytype,
+  t: any,
+): string {
+  switch (frequency) {
+    case Frequencytype.yearly:
+      return formattedPrice + t('price_per_year');
+    case Frequencytype.monthly:
+      return formattedPrice + t('price_per_month');
+    case Frequencytype.hourly:
+      return formattedPrice + t('price_per_hour');
+    case Frequencytype.daily:
+      return formattedPrice + t('price_per_day');
+    default:
+      return formattedPrice;
+  }
+}
+
+export function formatPrice(
+  price: number,
+  frequency: Frequencytype,
+  unit: CentsUnit,
+  t: any,
+  options?: PriceOptions,
+) {
+  const { htOnly, withTTC, includeGST, excludeGST, htWithTTC, ovhSubsidiary } =
+    options || {};
+
+  if (price === 0) {
+    return t('price_free');
+  }
+
+  const convertedPrice = getConvertedPrice(price, unit);
+  const roundedPrice = Math.round(convertedPrice * 100) / 100;
+  let formattedPrice = `${roundedPrice.toFixed(2).replace(/\.00$/, '') || ''}`;
+
+  if (ovhSubsidiary) {
+    formattedPrice = formatCurrency(formattedPrice, ovhSubsidiary);
+  }
+
+  if (htOnly) {
+    formattedPrice += t('price_ht_label');
+  }
+
+  if (includeGST) {
+    formattedPrice += t('price_gst_incl_label');
+  }
+
+  if (excludeGST) {
+    formattedPrice += t('price_gst_excl_label');
+  }
+
+  if (withTTC) {
+    formattedPrice += t('price_ttc_label');
+  }
+
+  if (htWithTTC) {
+    const priceHT = parseFloat(formattedPrice);
+    const priceTTC = priceHT * 1.2;
```

Comment:
are we sure it's always 20% ?

we should provide a way to change that by having a props for taxRate (defaulted to 20%)